hmda/src/main/scala/hmda/api/http/admin/InstitutionAdminHttpApi.scala

```diff
@@ -116,6 +79,53 @@ trait InstitutionAdminHttpApi extends HmdaTimeDirectives {
       }
     }
 
+  private def postInstitution(institution: Institution, uri: Uri): Route = {
+    val institutionPersistence = InstitutionPersistence.selectInstitution(sharding, institution.LEI, institution.activityYear)
```

Comment:
For this portion of the issue :
```
Check that no matching tax-id exists when making or updating an institution through the institutions-api.
```
Should we elevate tax-id to be a unique constraint as we do LEI?